### PR TITLE
Sine wave pH

### PR DESCRIPTION
**Pull request by eucalvo**

### CODE REVIEW

**Review by jgfoster — APPROVED** [review]

File: src/Devices/EEPROM_TC.cpp [hunk 1]
```diff
@@ -126,6 +135,9 @@ uint32_t EEPROM_TC::getRampTimeEndTemp() {
 float EEPROM_TC::getRampStartingTemp() {
   return eepromReadFloat(RAMP_STARTING_TEMP_ADDRESS);
 }
+uint16_t EEPROM_TC::getPHSetType() {
```
Comment (prestoncarman) [inline src/Devices/EEPROM_TC.cpp:138]: "Ph" or "PH"? Lets be consistent and make it always "Ph" in the function names.

File: src/Devices/EEPROM_TC.cpp [hunk 2]
```diff
@@ -213,6 +234,9 @@ void EEPROM_TC::setRampTimeEndTemp(uint32_t value) {
 void EEPROM_TC::setRampStartingTemp(float value) {
   eepromWriteFloat(RAMP_STARTING_TEMP_ADDRESS, value);
 }
+void EEPROM_TC::setPHSetType(uint16_t value) {
```
Comment (prestoncarman) [inline src/Devices/EEPROM_TC.cpp:237]: "Ph" or "PH"? Lets be consistent and make it always "Ph" in the function names.

File: src/Devices/EEPROM_TC.h [hunk 3]
```diff
@@ -26,16 +26,20 @@ class EEPROM_TC {
...
+  uint32_t getPhSinePeriod();
+  float getPhSineAmplitude();
   uint32_t getRampTimeStartTemp();
   uint32_t getRampTimeEndTemp();
   float getRampStartingTemp();
-  float getPHDelay();            // not used
-  float getPHInterval();         // not used
-  float getPHSeriesPointer();    // not used
-  float getPHSeriesSize();       // not used
+  float getPHDelay();          // not used
```
Comment (prestoncarman) [inline src/Devices/EEPROM_TC.h:38]: "Ph" or "PH"? Lets be consistent and make it always "Ph" in the function names. Update for the whole file.

File: test/MenuTest.cpp [hunk 4]
```diff
@@ -37,6 +37,11 @@ unittest_teardown() {
   enterKey('D');
 }
 
+// 2 goes up in the menu
```
Comment (prestoncarman) [inline test/MenuTest.cpp:40]: I think this comment would be nice to have as a multiline comment associated with the function.

```c++
/**
 * 2 goes up in the menu
 * ...
 */
unittest(MainMenu) {
```

This should help it to show up with function documentation.

**Review by prestoncarman — CHANGES_REQUESTED** [review]
The main question is how to deal with naming pH variables and files. I think we need to be more consistent. We seem to use "pH", "PH", and "Ph". I can see arguments for all of these, but in most cases I think that "Ph" is probably the correct answer.
Thought?

**Comment by jgfoster** [comment]
The chemists and biologists use "pH" as the proper designation, but that doesn't fit very well with camelCase. So, consistency is good and Ph fits so I'd be inclined to go with that...

**Review by prestoncarman — APPROVED** [review]

**Review by jgfoster — APPROVED** [review]

**Pull request merged**